_data/nav.yml

```diff
@@ -318,7 +318,7 @@
     - url: "anchor"
     - url: "autolink"
       pages:
-      - url: "#default_link_target"
+      - url: "#link_default_target"
```

Comment:
These need reordering as previously it was in alphabetical order.

- [ ] Reorder the sections so they remain in alphabetical order please